src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1132,6 +1132,17 @@ def get_state(self) -> Any:
 
     getState = get_state
 
+    def set_state(self, state) -> Any:
+        """Get state of the command argument."""
+        parent_state = self.parent.get_state()
+        try:
+            parent_state[self.name] = state
+            self.parent.set_state(parent_state)
+        except KeyError:
+            pass
```

Comment:
Question unrelated to this PR: why are we ignoring `KeyError` exceptions here? Should we at least log these?

Reply:
I think we should.

cc. @mkundu1, @seanpearsonuk

Reply:
Done.